app/src/main/java/com/example/gradeassist/dao/IQuestionDAO.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.gradeassist.dao
+
+import com.example.gradeassist.dto.Question
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface IQuestionDAO {
+
+    // Get JSON Data to display on UI...Not ready yet
+
+    //    @GET("https://drive.google.com/file/d/1EtNU3cXWl9e5_si-aoiZm_qXvHVN6OFw/view?usp=sharing")
+    fun getAllQuestions(): Call<ArrayList<Question>>
+
+    //    @GET("https://drive.google.com/file/d/1EtNU3cXWl9e5_si-aoiZm_qXvHVN6OFw/view?usp=sharing")
+    fun getQuestionById(questionId: Int): Call<Question>
```

Comment:
Added a new function to get a Question by ID

Reply:
Uncomment the @Get annotations and use actual endpoints make them work.  Otherwise, this feature is incomplete.

Reply:
the group never provided an actual endpoint, that's why i commented it out.